go.mod

```diff
@@ -0,0 +1,28 @@
+module github.com/gophish/gophish
```

Comment:
Please remove the `go.sum` and `go.mod` files before the final review. I'll add dependency management at some point, but that should be separate from the work here.

Reply:
Ok